### DISCUSSION TITLE
default language

### DISCUSSION THREAD

**Discussion by happyreacer** [nextcloud/all-in-one#482] [Questions] [answered]
How to set a language as default?

**Comment by szaimen** [comment]
You mean in Nextcloud? See https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/language_configuration.html for that

**Comment by happyreacer** [comment]
yes, in principle. but can i also parameterize this somewhere when starting the container?

**Comment by szaimen** [comment, marked as answer]
Based on https://github.com/nextcloud/all-in-one#how-to-run-occ-commands and https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/language_configuration.html you can set the locale and language as you wish after the container was started the first time:
E.g. `sudo docker exec --user www-data -it nextcloud-aio-nextcloud php occ config:system:set default_language --value="en"`

**Reply by happyreacer** [reply]
thanks a lot

**Comment by jakobroehrl** [comment]
I get this error

```
Current user id: 0
Owner id of config.php: 33
Try adding 'sudo -u #33' to the beginning of the command (without the single quotes)
If running with 'docker exec' try adding the option '-u 33' to the docker command (without the single quotes)
```

So, this works for me:
`sudo docker exec -u33 -it nextcloud-aio-nextcloud php occ config:system:set default_language --value="de"`
